static/elements/chromedash-feature.html

```diff
@@ -148,72 +180,156 @@
+      },
+      MAX_STANDARDS_VAL: {
+        type: Number,
+        value: 6,
+        readOnly: true
+      },
+      MAX_VENDOR_VIEW: {
+        type: Number,
+        value: 7,
+        readOnly: true
+      },
+      MAX_WEBDEV_VIEW: {
+        type: Number,
+        value: 6,
+        readOnly: true
+      },
+      compatRisk: {
+        type: Number,
+        value: 0
+      },
+      deprecated: {
+        type: Boolean,
+        computed: '_computeDeprecated(feature.impl_status_chrome)'
+      },
+      removed: {
+        type: Boolean,
+        computed: '_computeRemoved(feature.impl_status_chrome)'
+      },
+      feature: {
+        value: null,
+        observer: '_featureChanged'
+      },
+      needsflag: {
+        type: Boolean,
+        value: false
+      },
+      open: {
+        type: Boolean,
+        value: false,
+        notify: true,
+        observer: '_openChanged'
+      }
     },
-    toggle: function(e, details, sender) {
+
+    toggle: function(e, details) {
       // Don't toggle panel if tooltip or link is being clicked.
-      if (e.target.classList.contains('tooltip') ||
-          'tooltip' in e.target.dataset || e.target.tagName == 'A' ||
-           e.target.tagName == 'MULTI-LINKS') {
+      var target = Polymer.dom(e).localTarget;
+      if (target.classList.contains('tooltip') || 'tooltip' in target.dataset ||
+          target.tagName == 'A' || target.tagName == 'MULTI-LINKS') {
         return;
       }
       this.open = !this.open;
     },
-    calculateCompatRisk: function() {
-      var vendors = (this.feature.ff_views.value +
-                     this.feature.ie_views.value +
-                     this.feature.safari_views.value) / 3;
+
+    categoryFilter: function(e, details) {
+      e.stopPropagation();
+      this.fire('filter-category', {val: Polymer.dom(e).localTarget.textContent});
+    },
+
+    ownerFilter: function(e, details) {
+      e.stopPropagation();
+      this.fire('filter-owner', {val: Polymer.dom(e).localTarget.textContent});
+    },
+
+    _urlizeFilter: function(val) {
+      return urlize(val, {target: '_blank', trim: 'www', autoescape: true});
+    },
+
+    _calculateCompatRisk: function() {
+      var vendors = (this.feature.ff_views.value + this.feature.ie_views.value + this.feature.safari_views.value) / 3;
       var webdevs = this.feature.web_dev_views.value;
       var standards = this.feature.standardization.value;
-
       this.compatRisk = vendors + webdevs + standards;
     },
-    featureChanged: function() {
-      this.calculateCompatRisk();
 
-      // TODO: these values are brittle if the strings change.
-      this.removed = this.feature.impl_status_chrome == 'Removed';
-      this.deprecated = this.feature.impl_status_chrome == 'Deprecated' ||
-                        this.feature.impl_status_chrome == 'No longer pursuing';
+    _openChanged: function() {
+      this.open ? this.classList.add('open') : this.classList.remove('open');
```

Comment:
I'm not a fan of using the ternary operator as an if/else replacement while ignoring the return value.